puzzles/vault_puzzles/restriction_puzzles/wrappers/force_1_of_2_w_restricted_variable.clsp

```diff
@@ -0,0 +1,89 @@
+; This is a highly specific puzzle that enforces that any new create coins are a 1of2 w/ a fixed left side
+; and a variable right side that is put under pre-committed restrictions.
+;
+; The framework this is designed to fit in doesn't really support subtree behavior like this, so the specificity and
+; limited flexibility of this puzzle is due to the fact that it's sort of a hack to support an existing feature with
+; a framework poorly designed for it.
+(mod
+  (
+    DELEGATED_PUZZLE_FEEDER_MOD_HASH
+    1_OF_N_MOD_HASH
+    LEFT_SIDE_SUBTREE_HASH_HASH
+    INDEX_WRAPPER_HASH
+    NONCE
+    RESTRICTION_MOD_HASH
+    MEMBER_VALIDATOR_LIST_HASH
+    DPUZ_VALIDATOR_LIST_HASH
+    Conditions
+    new_right_side_member_hash
+  )
+
+  (include curry.clib)
+  (include utility_macros.clib)
+  (include condition_codes.clib)
+  (defconstant ONE 1)
+
+  (defun check_coins (conditions fixed_puzzle_hash)
+    (if conditions
+        (if (and (= (f (f conditions)) CREATE_COIN) (not (= (f (r (f conditions))) fixed_puzzle_hash)))
+            (x)  ; (f (r (f conditions))) fixed_puzzle_hash)
```

Comment:
I don't know why this comment is here, it's probably my fault, but we should delete it.

Reply:
Removed it